juju/juju
# Move to Pubsub v2

The following moves to pubsub v2, which doesn't require everyone to wait
for every subscriber to be done. Instead, if you want the old
functionality you can call the high-level function in the package.

## QA steps

```sh
$ juju bootstrap lxd test
$ juju enable-ha
$ juju deploy cs:~jameinel/ubuntu-lite-9 ubuntu
$ juju deploy cs:~jameinel/ubuntu-lite-9 ubuntux
$ juju deploy cs:~jameinel/ubuntu-lite-9 ubuntuz
$ juju remove-machine 1 -m controller
$ juju enable-ha
```

Check the raft logs to see if there are any errors.

```sh
juju model-config -m controller logging-config="<root>=INFO;juju.worker.lease.raft=TRACE;juju.core.raftlease=TRACE"
```
## Bug reference

TBA
